### ISSUE TITLE
figure out why the newer versions of open-vm-tools don't work with docker-machine

### ISSUE THREAD

**Issue by SvenDowideit** [rancher/os#2092]
**RancherOS Version: (ros os version)**

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**

**Comment by ryansch** [comment]
Is it also possible to disable open-vm-tools?  I've got the parallels tools in a container configured to start on boot.

**Comment by SvenDowideit** [comment]
yup: http://rancher.com/docs/os/v1.1/en/configuration/adding-kernel-parameters/#enabledisable-hypervisor-service-auto-enable

**Comment by SvenDowideit** [comment]
Do you have a link for that image and service yml? iirc, we detect parralells, so we can add it to the rancher/os-services and have magic.

**Comment by ryansch** [comment]
@SvenDowideit https://github.com/outstand/docker-parallels-tools

I don't bother with the parallels file sharing discussed in the README these days.

**Comment by Darkknight33** [comment]
I want to add that this is currently causing an blocking issue with deploying RancherOS on ESXi hosts. The tools don't install properly, and has at least once out of 5 attempts caused the VM to become unresponsive (docker daemon crashed) and auto-deleted by Rancher for failing to provision correctly. Interrupting it with CTRL-C right when it gets stuck will get you back to the CLI. I tried running rancher/os-openvmtools:10.0.7.52125 (e.g. docker run -d --name openvmtools rancher/os-openvmtools:10.0.7.52125) but it fails with: 

docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"/usr/bin/ros\": stat /usr/bin/ros: no such file or directory"
